pkgs/by-name/sy/sysdig-cli-scanner/package.nix

```
@@ -0,0 +1,54 @@
+{
+  stdenv,
+  lib,
+  fetchurl,
+  makeWrapper,
+}:
+let
+  versionMetadata = builtins.fromTOML (builtins.readFile ./sysdig-cli-scanner.versions.toml);
+  fetchForSystem = versionMetadata.${stdenv.system} or (throw "unsupported system ${stdenv.system}");
+in
+stdenv.mkDerivation {
+  pname = "sysdig-cli-scanner";
+  version = versionMetadata.version;
+
+  src = fetchurl { inherit (fetchForSystem) url hash; };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  dontUnpack = true;
+
+  installPhase = ''
+    runHook preInstall
+
+    install -D $src $out/bin/sysdig-cli-scanner
+
+    wrapProgram $out/bin/sysdig-cli-scanner \
+      --add-flags --dbpath="/tmp/sysdig-cli-scanner/"
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "A versatile tool for scanning container images and directories using Sysdig";
```

According to the [recommendations for meta attributes](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes) description should
* not start with an article
* avoid subjective language
```suggestion
    description = "Tool for scanning container images and directories using Sysdig";
```

---

Thanks for the information @afh, I will change this with your suggestion and I will squash all the commits in the PR so it's cleaner.